Remove and Update topics.

MichelDiz opened the pull request:
- Remove "expand and _predicate_"
>We already have a "Expand" topic and "`_predicate_`" was removed from Dgraph core.
- Update topic according to the new Type System.
- Remove "`_predicate_`" example

MichelDiz committed:
Remove "_predicate_" example

MichelDiz committed:
Update topic according to the new Type System.

MichelDiz committed:
Remove "expand and _predicate_"

We already have a "Expand" topic and "_predicate_" was removed from Dgraph core.

mangalaman93 commented on content/moredata/2.md:
```diff
@@ -4,25 +4,31 @@ next = "/blocksvars/1"
 Let's have a look from the perspective of directors.  Kathryn Bigelow directed the early 90's classic Point Break.  We'll begin with her and learn about the schema.
 
-Run the query.  `_predicate_` tells us that directors have a `name` and are connected to the movies they directed via `director.film`.
+Checking the Type Schema, it tells us that directors as a "Person Type" have a `name` and are connected to the movies they directed via `director.film`.
```
Checking the Type Schema tells us

mangalaman93 commented on content/moredata/2.md:
````diff
@@ -4,25 +4,31 @@ next = "/blocksvars/1"
 Extend the query a level deeper by also including
 ```
-director.film (first: 1) { _predicate_ }
+director.film (first: 1) {     
````
We should query the type schema here.

MichelDiz replied:
But this isn't a query. Is an edge (a nested block). Do you think I should revamp this part as a query?

MichelDiz replied:
I'll merge and solve this after.

mangalaman93 commented on content/moredata/2.txt:
```diff
@@ -1,6 +1,26 @@
+    performance.actor: [Person]
+    performance.special_performance_type: [Special_performance_type]
+    type: [Generic]
+}
```
Newline missing

MichelDiz committed:
Add Aman's requests and other modifications.

I've changed the link to download 1million.rdf.gz - I don't get why we have multiple files as benchmarks repo can be our source.

MichelDiz committed:
Removed old file and undo download url change.
